doc/guides/dts/intro.rst

```diff
@@ -121,11 +131,106 @@ names and properties you might see when working with I2C devices.
 .. figure:: zephyr_dt_i2c_example.png
    :figclass: align-center
 
-   I2C devicetree example with real-world names and properties
+   I2C devicetree example with real-world names and properties.
+   Node names are at the top of each node with a gray background.
+   Properties are shown as "name=value" lines.
+
+This is the corresponding DTS:
+
+.. code-block:: DTS
+
+   /dts-v1/;
+
+   / {
+           soc {
+                   i2c@40003000 {
+                           compatible = "nordic,nrf-twim";
+                           label = "I2C_0";
+                           reg = <0x40003000 0x1000>;
+
+                           apds9960@39 {
+                                   compatible = "avago,apds9960";
+                                   label = "APDS9960";
+                                   reg = <0x39>;
+                           };
+                           ti_hdc@43 {
+                                   compatible = "ti,hdc", "ti,hdc1010";
+                                   label = "HDC1010";
+                                   reg = <0x43>;
+                           };
+                           mma8652fc@1d {
+                                   compatible = "nxp,fxos8700", "nxp,mma8652fc";
+                                   label = "MMA8652FC";
+                                   reg = <0x1d>;
+                           };
+                   };
+           };
+   };
+
+.. _dt-unit-address:
+
+In addition to showing more realistic names and properties, the above example
+introduces a new devicetree concept: unit addresses. Unit addresses are the
+parts of node names after an "at" sign (``@``), like ``40003000`` in
+``i2c@40003000``, or ``39`` in ``apds9960@39``. Unit addresses are optional:
+the ``soc`` node does not have one.
+
+Some more details about unit addresses and important properties follow.
+
+Unit address examples
+*********************
 
-Above, node names -- like ``i2c@40003000`` -- are at the top of each node, with
-a gray background, except for the root node, which is shown using its path
-``/``. Properties are shown as ``name=value`` pairs below the node names.
+In devicetree, unit addresses give a node's hexadecimal address in the
```

Comment:
While the unit address has typically been a hexadecimal address this isn't required by the devicetree spec.  And specifically if we properly support PCI we could have something like `pci@1,0`

Reply:
Good to know, thank you!